### PR TITLE
[BLOCKER] Adding 4-action-page template, questions include. closes #2099

### LINKED ISSUE #2098: Create Questions include
Make questions box into an include, and optionally use Front Matter data instead. [See example on right side of this page](https://www.vets.gov/disability-benefits/conditions/exposure-to-hazardous-materials/gulf-war-illness/) 

Blocks #2089. Related to #2099

### LINKED ISSUE #2099: Create 4-action-page template
Create a `4-action-page` with a call out box, and optional Questions box.

**Example pages**
- https://www.vets.gov/education/work-learn/job-and-apprenticeship/
- https://www.vets.gov/disability-benefits/conditions/exposure-to-hazardous-materials/gulf-war-illness/

Similar to #2097, #2101. Related to #2098. Blocks #2089.

### PR DESCRIPTION
**Pull request by webinista**
- Closes #2098
- Deletes two pages that contained "coming soon" text:
  - _disability-benefits/conditions/exposure-to-hazardous-materials/radiogenic-risk-activities/associated-diseases.md
  - _education/gi-bill/post-9-11/fry-scholarship.md
- Updated template YAML values for many pages
- Removed markup from pages tagged with template: 4-action-page
- Blocks #2089.
- Blocks #2106.
- Blocks #2366

### CODE REVIEW

File: _disability-benefits/conditions/exposure-to-hazardous-materials/project112-SHAD.md [hunk 1]
```diff
@@ -1,28 +1,11 @@
 ---
```
Comment (emilyville) [inline _disability-benefits/conditions/exposure-to-hazardous-materials/project112-SHAD.md]: this page is broken

File: _disability-benefits/conditions/exposure-to-hazardous-materials/radiogenic-risk-activities.md [hunk 2]
```diff
@@ -1,85 +1,36 @@
 ---
```
Comment (emilyville) [inline _disability-benefits/conditions/exposure-to-hazardous-materials/radiogenic-risk-activities.md]: broken too!

File: _disability-benefits/conditions/exposure-to-hazardous-materials/specific-environmental-hazards.md [hunk 3]
```diff
@@ -1,69 +1,32 @@
 ---
```
Comment (emilyville) [inline _disability-benefits/conditions/exposure-to-hazardous-materials/specific-environmental-hazards.md]: here too!
https://vets-website-pr-2410.herokuapp.com/disability-benefits/conditions/exposure-to-hazardous-materials/specific-environmental-hazards/

File: _education/gi-bill/higher-learning.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (emilyville) [inline _education/gi-bill/higher-learning.md]: this page has a markup error somewhere with a stray div at the bottom. (it's a pre-existing issue, so could be fixed in another PR)

**Comment by emilyville** [comment]
Looks great! A few pages look a little wonky, kicking it back to you!

**Comment by webinista** [comment]
Got my branches mixed up and ended up adding a template name that wasn't actually added to this PR. Should be fixed now. Just added the markup that was removed. Will update that change in another pull request.

**Comment by emilyville** [comment]
LGTM!

**Pull request merged**